### PR TITLE
Feature/unloadable vobs fix

### PR DESCRIPTION
**Pull request by JaXt0r**
What changed:
* Freepoints are now visible if activated on DebugSettings (round balls)
* Decals are now imported
* Shader moved from Unlit to URP/Unlit - Please check mobile performance

Can be tested now, but merged after https://github.com/GothicKit/phoenix-csharp-interface/pull/17 only.

---

# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene
* [x] FeatureFlags reverted to right value

# Testing
* [x] Merged _main_ into this branch and tested with the latest features
* [ ] Tested with PCVR and one mobile VR device (Pico/Quest)

## Dependencies
* [x] If libphoenix-shared.dll is changed, .so is also changed

### CODE REVIEW

File: Assets/GothicVR/Scripts/Creator/VobCreator.cs [hunk 1]
```diff
@@ -164,11 +196,76 @@ private void CreateZoneMusic(PxVobZoneMusicData vob)
...
+            SetPosAndRot(spot, vob.position, vob.rotation!.Value);
+        }
+
         private GameObject CreateItemMesh(PxVobItemData vob, PxVmItemData item, GameObject go)
         {
             var mrm = assetCache.TryGetMrm(item.visual);
             return meshCreator.Create(item.visual, mrm, vob.position.ToUnityVector(), vob.rotation!.Value, true, parentGos[vob.type], go);
         }
+
+        private void CreateDecal(PxVobData vob)
```
Comment (JucanAndreiDaniel) [inline Assets/GothicVR/Scripts/Creator/VobCreator.cs:235]: This function should be moved to MeshCreator

File: Assets/GothicVR/Scripts/Creator/VobCreator.cs [hunk 2]
```diff
@@ -164,11 +196,76 @@ private void CreateZoneMusic(PxVobZoneMusicData vob)
             soundCreator.Create(vob, parentGos[vob.type]);
         }
 
+        /// <summary>
+        /// Basically a free point where NPCs can do something like sitting on a bench etc.
+        /// @see for more information: https://ataulien.github.io/Inside-Gothic/objects/spot/
+        /// </summary>
+        private void CreateSpot(PxVobData vob)
```
Comment (JucanAndreiDaniel) [inline Assets/GothicVR/Scripts/Creator/VobCreator.cs:203]: Add Debug Option to enable/disable the spots from rendering

File: Assets/GothicVR/Scripts/Creator/VobCreator.cs [hunk 3]
```diff
@@ -4,22 +4,28 @@
...
+    using UnityEditor;
+#endif
+
+
 namespace GVR.Creator
 {
 	public class VobCreator : SingletonBehaviour<VobCreator>
-    {   
+    {
+        private const float decalOpacity = 0.5f;
```
Comment (JucanAndreiDaniel) [inline Assets/GothicVR/Scripts/Creator/VobCreator.cs:27]: decals should be 1f most of the time, exception should be maybe dirt on castle walls

File: Assets/GothicVR/Scripts/Creator/MeshCreator.cs [hunk 4]
```diff
@@ -471,5 +471,16 @@ private static void CreateBonesData(GameObject root, SkinnedMeshRenderer rendere
             renderer.sharedMesh.bindposes = bindPoses;
             renderer.bones = bones;
         }
+        
+        private Material GetEmptyMaterial()
+        {
+            var standardShader = Shader.Find(defaultShader);
```
Comment (JucanAndreiDaniel) [inline Assets/GothicVR/Scripts/Creator/MeshCreator.cs:477]: Check if defaultShader is loaded correctly as right now the shader cannot be found on builds and returns null

**Review by JucanAndreiDaniel — CHANGES_REQUESTED** [review]
Mostly small changes requested :D

File: Assets/GothicVR/Scripts/Creator/MeshCreator.cs (hunk 4 above)
Comment (JaXt0r) [reply Assets/GothicVR/Scripts/Creator/MeshCreator.cs:477]: Found a way to reference it: https://stackoverflow.com/questions/36909567/textures-loads-in-editor-but-not-in-standalone-appears-pink

File: Assets/GothicVR/Scripts/Creator/VobCreator.cs (hunk 3 above)
Comment (JaXt0r) [reply Assets/GothicVR/Scripts/Creator/VobCreator.cs:27]: I would disagree. We discussed within Discord and mentioned 0.5f-0.75f
At least the walls look fine with this value and I would not want to have it "adjusted" for specific decals as of now.

Suggestion: Let's stick with a value and optimize it once we have lightning done right.

File: Assets/GothicVR/Scripts/Creator/VobCreator.cs (hunk 2 above)
Comment (JaXt0r) [reply Assets/GothicVR/Scripts/Creator/VobCreator.cs:203]: It's there. Just look into the method. It even adds labels if we're in Editor mode.

Comment (JaXt0r) [reply Assets/GothicVR/Scripts/Creator/VobCreator.cs:203]: Added comment for better visibility.

**Comment by JaXt0r** [comment]
@JucanAndreiDaniel  Commented and solved. Please review again.

**Review by JucanAndreiDaniel — CHANGES_REQUESTED** [review]
Maybe Universal Render Pipeline/Unlit is not the best solution for now, we could only leave the submeshes to use Universal Render Pipeline/Unlit and the VOB to use Unlit/Transparent Cutout

File: Assets/GothicVR/Scripts/Creator/MeshCreator.cs [hunk 5]
```diff
@@ -471,5 +509,16 @@ private static void CreateBonesData(GameObject root, SkinnedMeshRenderer rendere
...
+            var material = new Material(standardShader);
+            
+            // Enable clipping of alpha values.
+            material.EnableKeyword("_ALPHATEST_ON");
```
[the remark this answers is not in the document]
Comment (JucanAndreiDaniel) [reply Assets/GothicVR/Scripts/Creator/MeshCreator.cs:519]: My bad, the issue was actually generated by not adding the decal shader which stops unity from loading the rest of the assets.

**Review by JucanAndreiDaniel — APPROVED** [review]
LGTM now, tested standalone quest and pcvr

**Pull request merged**